projects/ROCKNIX/packages/wayland/compositor/sway/autostart/111-sway-init

```diff
@@ -99,14 +99,16 @@ if hexdump -C /sys/class/drm/card0-DP-1/edid | grep -q "Lontium"; then
   echo "output DP-1 bg #000000 solid_color" >> $SWAY_HOME/config
   echo "output DP-1 allow_tearing yes"  >> $SWAY_HOME/config
   echo "output DP-1 max_render_time off" >> $SWAY_HOME/config
-  echo "output DP-1 scale 2" >> $SWAY_HOME/config
+  echo "output DP-1 scale 1" >> $SWAY_HOME/config
   echo "output DP-1 position 0 0" >> $SWAY_HOME/config
   echo "input \"8746:1:RetroidPocket_RDS_Touchscreen\" map_to_output DP-1" >> $SWAY_HOME/config
   echo "input \"8746:1:RetroidPocket_RDS_Touchscreen\" calibration_matrix 0 1 0 -1 0 1" >> $SWAY_HOME/config
 fi
 
 # Move the secondary window to the second output
-if [ "${DEVICE_HAS_DUAL_SCREEN}" = "true" ]; then
+if [ "${DEVICE_HAS_DUAL_SCREEN}" = "true" ] && hexdump -C /sys/class/drm/card0-DP-1/edid | grep -q "Lontium"; then
+  echo 'for_window [title=".*(Secondary|\[w2\]|Sub|Bottom|Screen 2|GamePad).*"] move window to output '"${connected_cons[1]}" >> $SWAY_HOME/config
+elif [ "${DEVICE_HAS_DUAL_SCREEN}" = "true" ]; then
   second_con=$([[ "${connected_cons[0]}" == "$con" ]] && echo "${connected_cons[1]}" || echo "${connected_cons[0]}")
   echo 'for_window [title=".*(Secondary|\[w2\]|Sub|Bottom|Screen 2|GamePad).*"] move window to output '"${second_con}" >> $SWAY_HOME/config
 fi
```

Comment:
```suggestion
if [ "$DEVICE_HAS_DUAL_SCREEN" = "true" ]; then
  if grep -qa "Lontium" /sys/class/drm/card0-DP-1/edid 2>/dev/null; then
    target_output="${connected_cons[1]}"
  else
    if [ "${connected_cons[0]}" = "$con" ]; then
      target_output="${connected_cons[1]}"
    else
      target_output="${connected_cons[0]}"
    fi
  fi

  window_regex='.*(Secondary|\[w2\]|Sub|Bottom|Screen 2|GamePad).*'
  echo "for_window [title=\"$window_regex\"] move window to output $target_output" >> "$SWAY_HOME/config"
fi
```

I think this would be better